.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Can you remove this file? :D

Reply:
It's gone in the next commit. Accidentally did a `git add .`.